app/src/main/java/comp3350/mealbuddy/objects/Account.java

```diff
@@ -0,0 +1,22 @@
+package comp3350.mealbuddy.objects;
+
+import java.util.ArrayList;
+
+public class Account {
+
+    public UserInfo user;
+    private ArrayList<Goal> goals;
+    public Account(UserInfo user) {
+        this.user = user;
+    }
+
+    public void addGoal(Goal g) {
+        if(goals.contains(g))
```

Comment:
Cant add new goals?
Also should we make an .equals() for goals that if they track the same 'thing' then they are equal?

Reply:
this was a typo, I fixed it. Not sure about how we want to go about goals, whatever you think is best.

Reply:
I think we want the equality operator in goals just to compare their id to make the most sense.  Doesn't make sense to have two goals tracking the same thing

Reply:
could be done in seperate pr or you can do it here